netlify.toml

```
@@ -78,3 +78,19 @@ to = "/v1/references/benchmarks"
 [[redirects]]
 from = "/references/*"
 to = "/v1/references/:splat"
+
+[[redirects]]
```

```suggestion
[[redirects]]
from = "/v1/api/js/classes/window.webviewwindow/*"
to = "/v1/api/js/window"

[[redirects]]
```

That's one of the most linked js pages so we should consider adding a specific redirect at least for that one.